### PR TITLE
Update callout mediator catalog

### PR DESCRIPTION
**Pull request by gabilang**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/reference/mediators/callout-mediator.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ The below example uses a Callout mediator to set a HTTP method when invoking a R
                <arg evaluator="xml" expression="$ctx:res"/>
             </args>
          </payloadFactory>
-         <send/>
+         <call/>
```
Comment (chanikag) [inline en/docs/reference/mediators/callout-mediator.md:204]: Let's use respond mediator. Please test it and update.

File: en/docs/reference/mediators/callout-mediator.md [hunk 2]
```diff
@@ -141,7 +141,7 @@ Following examples demonstrate the usage of the Callout mediator.
 
 In this example, the Callout Mediator does the direct service invocation to the `StockQuoteService` using the client request, gets the response, and sets the response as the first child of the SOAP message body. You can then use the [Send Mediator]({{base_path}}/reference/mediators/send-mediator) to send the message back to the client.
```
Comment (chanikag) [inline en/docs/reference/mediators/callout-mediator.md:142]: Change this according to the mediator you use in the sample

File: en/docs/reference/mediators/callout-mediator.md (hunk 1 above)
Comment (gabilang) [reply en/docs/reference/mediators/callout-mediator.md]: Modified the Xml script accordingly.

File: en/docs/reference/mediators/callout-mediator.md [hunk 3]
```diff
@@ -117,7 +117,7 @@ The parameters available for configuring the Callout mediator are as follows.
 <tr class="even">
 <td><strong>WS-Security</strong></td>
 <td><div class="content-wrapper">
-<p>If you select the check box, WS-Security is enabled for the Callout mediator. This section would expand as shown below when you select this check box.</p>
+<p>If you select the checkbox, WS-Security is enabled for the Callout mediator. This section would expand as shown below when you select this check box.</p>
```
Comment (chanikag) [inline en/docs/reference/mediators/callout-mediator.md:120]:
```suggestion
<p>If you select the checkbox, WS-Security is enabled for the Callout mediator. This section will expand as shown below when you select this check box.</p>
```

Comment (chanikag) [inline en/docs/reference/mediators/callout-mediator.md:120]: Is there a checkbox now in vs code?

File: en/docs/reference/mediators/callout-mediator.md [hunk 4]
```diff
@@ -60,7 +60,7 @@ The parameters available for configuring the Callout mediator are as follows.
 <li><strong>URL</strong> : Select <strong>URL</strong> if you want to call the external service by specifying its URL in the Call mediator configuration.</li>
 <li><strong>Address Endpoint</strong>: Select <strong>Address Endpoint</strong> if you want to call the external service via an <b>Endpoint</b>, which is already saved in the <b>Registry</b>. This option should be selected if you want to make use of the WSO2 functionality related to endpoints such as format conversions, security etc. Note that only Leaf endpoint types (i.e. <code>                Address               </code>, <code>WSDL</code>, <code>Default</code> and <code>Http</code>) are supported for the Callout mediator.</li>
 </ul>
-<p>If neither a URL or an address endpoint is specified, the <code>To</code> header on the request is used as the target endpoint.</p>
+<p>If neither a URL nor an address endpoint is specified, the <code>To</code> header on the request is used as the target endpoint.</p>
 </div></td>
```
Comment (chanikag) [inline en/docs/reference/mediators/callout-mediator.md:64]: The SOAP action which should be appended to the service call
->
The SOAP action should be appended to the service call

File: en/docs/reference/mediators/callout-mediator.md [hunk 5]
```diff
@@ -158,53 +158,44 @@ In this example, the Callout Mediator does the direct service invocation to the
...
+    <target>
+        <inSequence>
+            <property name="enableREST" scope="axis2-client" type="BOOLEAN" value="true"/>
+            <property name="HTTP_METHOD" scope="axis2-client" type="STRING" expression="$axis2:HTTP_METHOD"/>
+            <callout serviceURL="http://localhost:8280/callout/CalloutRESTApi">
+                <source type="envelope"/>
+                <target key="response"/>
+            </callout>
+            <log category="INFO" level="custom">
+                <property name="MESSAGE###########################3" expression="$ctx:response"/>
```
Comment (chanikag) [inline en/docs/reference/mediators/callout-mediator.md:175]: Let's change this log message to something meaningful

File: en/docs/reference/mediators/callout-mediator.md (hunk 3 above)
Comment (gabilang) [reply en/docs/reference/mediators/callout-mediator.md:120]: It seems that it has been changed to `true` or `false`.

Comment (gabilang) [reply en/docs/reference/mediators/callout-mediator.md]: Addressed with https://github.com/wso2/docs-mi/pull/538/commits/629f4f40357f33f655d0f4dac75b53a757e7602f

**Review by chanikag — APPROVED** [review]

**Pull request merged**